verifiers/internal/gha/provenance.go

```diff
@@ -315,7 +338,25 @@ func VerifyProvenance(env *dsselib.Envelope, provenanceOpts *options.ProvenanceO
 		if err := isValidDelegatorBuilderID(prov); err != nil {
 			return err
 		}
-		// Note: `provenanceOpts.ExpectedBuilderID` is provided by the user.
+
+		// If ExpectedID is not provided, check to see if it is a trusted builder.
+		// If not provided, then a trusted builder is expected, to populate provenanceOpts.ExpectedBuilderID
+		// with that builder, otherwise, populate from user input.
+		//
+		// Example: the expectedBuilderID will default to the delegator builder ID for BYOB.
```

Comment:
Which `expectedBuilderID` is meant here? I don't see one in this context? Do you mean the `provenanceOpts.ExpectedBuilderID` or

Reply:
That comment I now realize is misleading. Before that conditional, if there was no builder-id inputted, it would think the builder-id was the delegator. I was trying to show the functionality of that conditional but now I see it is just confusing! Let's remove that line.

Reply:
Removed: "// Example: the expectedBuilderID will default to the delegator builder ID for BYOB."